---a/net/minecraft/world/biome/BiomeGenMushroomIsland.java
+++b/net/minecraft/world/biome/BiomeGenMushroomIsland.java
@@ -3,12 +3,12 @@
 import net.minecraft.entity.passive.EntityMooshroom;
 import net.minecraft.init.Blocks;
 
-public class BiomeGenMushroomIsland extends BiomeGenBase
-{
+public class BiomeGenMushroomIsland extends BiomeGenBase {
+
     private static final String __OBFID = "CL_00000177";
 
-    public BiomeGenMushroomIsland(int p_i1984_1_)
-    {
+    public BiomeGenMushroomIsland(int p_i1984_1_) {
         super(p_i1984_1_);
         this.theBiomeDecorator.treesPerChunk = -100;
         this.theBiomeDecorator.flowersPerChunk = -100;
@@ -21,4 +21,4 @@
         this.spawnableWaterCreatureList.clear();
         this.spawnableCreatureList.add(new BiomeGenBase.SpawnListEntry(EntityMooshroom.class, 8, 4, 8));
     }
-}
+}
